thoth/storages/graph/postgres.py

```diff
@@ -3606,6 +3607,65 @@ def _runtime_environment_conf2models(
 
         return hardware_information, software_environment
 
+    def create_github_app_installation(self, slug: str, repo_name: str, private: bool, installation_id: str) -> bool:
+        """Create a record for new installation or reactivate uninstalled installation.
+
+        Example -
+            "installation_id": "236796147",
+            "repo_name": "advisor",
+            "slug": "thoth-station/advisor",
+            "private": False
+
+        Accepts installation details passed down by Github.
+        :rtype: True, False
+        :raises True: if installation existed and was updated
+        :raises False: if installation was newly added.
+        """
+        with self._session_scope() as session:
+            instance = session.query(KebechetGithubAppInstallations)\
+                .filter(KebechetGithubAppInstallations.slug == slug).first()
+            if instance:
+                instance.installation_id = installation_id
+                instance.private = private
+                is_active = True
+                session.commit()
+                return True
+            else:
+                _, newly_added = KebechetGithubAppInstallations.get_or_create(
+                    session,
+                    slug=slug,
+                    repo_name=repo_name,
+                    private=private,
+                    installation_id=installation_id,
+                    is_active=True
+                )
+                return newly_added
+
+    def deactivate_github_app_installation(self, slug: str) -> bool:
+        """We deactivate the app on getting an uninstall event.
+
+        Passed a slug name to be deactivated.
+        Example - slug:'thoth-station/advisor'
+        :rtype: True, False
+        :raises True: if installation existed and was deactivated.
+        :raises False: if installation was not found.
+        """
+        with self._session_scope() as session:
+            instance = session.query(KebechetGithubAppInstallations)\
+                .filter(KebechetGithubAppInstallations.slug == slug).first()
+            if instance:
+                instance.is_active = False
+                session.commit()
+                return True
+            return False
+
+    def get_active_kebechet_installations(self) -> int:
```

Comment:
please follow https://github.com/thoth-station/storages/blob/master/docs/conventions/README.md convetions for query name :)

Reply:
`get_kebechet_github_installations_count_per_is_active` this is what I could derive, is that right? :smiley: